jobs/user_add/templates/pre-start.sh.erb

```diff
@@ -25,10 +32,12 @@ set -ex
       end
       %>
 
-  getent passwd <%=user%> > /dev/null || useradd --create-home <%=user%>
+  getent passwd <%=user%> > /dev/null || useradd --create-home <%=user%> --base-dir "$home_dir"
   passwd -d <%=user%>
 
-  <% unless user_hash['shell'].nil? %>
+  <% if user_hash['shell'].nil? %>
+    usermod -s /bin/bash <%=user%>
```

Comment:
why do we need this one?

Reply:
Because when I tested it, the shell wasn't set to bash but sh as opposed to what is pointed out in the readme. I used stemcell 3445.2 but it happened also on an earlier one (don't remember right now what version it was).

Reply:
Interesting - originally we avoided configuring a default shell in favor of relying on the operating system's default shell (e.g. when centos/ubuntu have different settings). There was an assumption that this would typically be `bash` though. Hmm.

```
$ grep ^SHELL /etc/default/useradd
SHELL=/bin/sh
```

Reply:
Actually my first approach was setting SHELL in /etc/default/useradd to bash at the beginning of the pre-start.sh script but it was ignored. Didn't want to investigate so I switched to usermod...